doc/api/deprecations.md

```diff
@@ -3774,6 +3774,19 @@ Type: Runtime
 
 The `util.getCallSite` API has been removed. Please use [`util.getCallSites()`][] instead.
 
+### DEP0187: `fs.existsSync` never throw on invalid arguments
+
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/55753
+    description: Runtime deprecation.
+-->
+
+Type: Runtime
+
+Returning false on validation errors is deprecated.
```

Comment:
We deprecate the user behavior, not the runtime one. In the future, if that deprecation cycle goes to EoL, passing an invalid argument will throw an error
```suggestion
Passing invalid argument types is deprecated.
```